examples/next-prisma-starter/tsconfig.json

```
@@ -18,18 +14,13 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "types": [
-      "@types/jest",
-    ],
-    "baseUrl": "./src",
-    "incremental": true
+    "types": ["@types/jest"],
+    "incremental": true,
+    "baseUrl": ".",
+    "paths": {
+      "~/*": ["./src/*"]
```

```suggestion
      "~/*": ["src/*"]
```
not needed when `baseUrl` is already `.`.

---

Fixed